WordPress/Classes/Stores/NoticeStore.swift

```diff
@@ -5,8 +5,11 @@ import WordPressFlux
 /// the app, much like Android toasts or snackbars.
 /// Once you've created a Notice, you can dispatch a `NoticeAction` to display it.
 ///
-struct Notice {
+struct Notice: Equatable {
```

Comment:
I think it's probably personal preference rather than something in our style guide, but I would usually declare protocol conformance in an extension rather than the main struct definition (the implementation of the protocol then stays next to the declaration).

Reply:
That makes sense. Thanks for pointing it out.